src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -43,14 +42,6 @@ class AnnotationLoader implements LoaderInterface
         Context::class,
     ];
 
-    public function __construct(
-        private readonly ?Reader $reader = null,
-    ) {
-        if ($reader) {
-            trigger_deprecation('symfony/validator', '6.4', 'Passing a "%s" instance as argument 1 to "%s()" is deprecated, pass null or omit the parameter instead.', get_debug_type($reader), __METHOD__);
-        }
-    }
```

Comment:
Question: should I keep an empty constructor so we won't break overrides that call `parent::__construct()`?

Reply:
no need to IMHO